src/Core/templates/Maker/Backend/Actions/Index.tpl.php

```diff
@@ -0,0 +1,17 @@
+<?= "<?php\n"; ?>
+
+namespace <?= $namespace ?>;
+
+use ForkCMS\Modules\Backend\Domain\Action\AbstractDataGridActionController;
```

Comment:
Should have a use statement for Symfony\Component\HttpFoundation\Request too

Reply:
fixed in 08ab4d0dfdaa75e040719482974ec104c510c05d